misc/config_tools/schema/config.xsd

```diff
@@ -99,27 +104,28 @@
       </xs:annotation>
     </xs:element>
     <xs:element name="ACPI_PARSE_ENABLED" type="Boolean" default="y">
-      <xs:annotation acrn:title="Parse ACPI tables at runtime" acrn:views="advanced">
+      <xs:annotation acrn:title="Parse ACPI tables" acrn:views="advanced">
         <xs:documentation>Enable ACPI runtime parsing to get DMAR (DMA remapping) configuration data from the APCI tables. Otherwise, use existing, static information from the associated board configuration file.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="L1D_VMENTRY_ENABLED" type="Boolean" default="y">
       <xs:annotation acrn:title="Mitigate L1 terminal fault" acrn:views="advanced">
-        <xs:documentation>Enable L1 cache flush before VM entry to prevent L1 terminal fault. L1 terminal fault is a hardware vulnerability that allows unauthorized disclosure of information residing in the L1 data cache.</xs:documentation>
+        <xs:documentation>Enable L1 cache flush before VM entry to prevent L1 terminal fault. L1 terminal fault is a hardware vulnerability that could allow unauthorized disclosure of information residing in the L1 data cache.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="MCE_ON_PSC_DISABLED" type="Boolean" default="y">
-      <xs:annotation acrn:title="Machine Check Error (MCE) workaround" acrn:views="advanced">
-        <xs:documentation>Enable the software workaround for Machine Check Error on Page Size Change (hardware bug in some processor families).</xs:documentation>
+      <xs:annotation acrn:title="MCE workaround" acrn:views="advanced">
+        <xs:documentation>Enable the software workaround for Machine Check Error on Page Size Change (erratum in some processor families).</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="IVSHMEM" type="IVSHMEMInfo">
       <xs:annotation acrn:title="Inter-VM shared memory" acrn:views="basic">
-        <xs:documentation/>
+        <xs:documentation>Configure shared memory regions for inter-VM communication.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="SSRAM" minOccurs="0" type="SSRAMInfo">
-      <xs:annotation acrn:title="Software SRAM (for real-time apps)" acrn:views="advanced">
+      <xs:annotation acrn:title="Software SRAM" acrn:views="advanced">
+        <xs:documentation>Configure Software SRAM. This feature reserves memory buffers as always-cached memory to improve an application's real-time performance.</xs:documentation>
         <xs:documentation />
```

Comment:
@amyreye Because you added a new documentation child, you need to delete this line that now adds a second (empty) documentation child.